Sources/MongoSwift/Operations/DropIndexesOperation.swift

```diff
@@ -14,24 +14,29 @@ public struct DropIndexOptions: Encodable {
 /// An operation corresponding to a "dropIndexes" command.
 internal struct DropIndexesOperation<T: Codable>: Operation {
     private let collection: MongoCollection<T>
+    private let session: ClientSession?
```

Comment:
this is a bit out of scope for this work, but I imagine the operations, when we do client pooling, will try to get a collection via the session first, then from the stored mongo collection second if there is no stored session.

Reply:
I think we should only ever need a namespace, no need to eventually store a collection or a database